packages/stdlib/test.pony

```diff
@@ -62,6 +63,7 @@ actor Main is TestList
     logger.Main.make().tests(test)
     net.Main.make().tests(test)
     options.Main.make().tests(test)
+    process.Main.make().tests(test)
```

Comment:
this doesnt work on windows so it should probably be in an `ifdef not windows` block